app/index.html

```diff
@@ -33,7 +33,7 @@
       </div>
     </div>
     <div id="panel-toggle">
-      <button id="open-toggle" title="Open properties panel">⟨</button>
+      <button id="open-toggle" title="Open properties panel"><span>⟨</span></button>
```

Comment:
Why not put the other one in a `span`, too?

\edit: Also, unfortunately, it does not seem to fix the error in my Firefox 68.5.0esr (64-bit) :no_mouth: